.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: C
+
+install:
+  - CHANNEL=http://eupsforge.net/conda/dev ./setup/install.sh lsst-sims
```

Comment:
I'd propose to change this to:
```
CHANNEL=http://eupsforge.net/conda/dev ./setup/install.sh lsst-sims nose pandas
```
That way you don't need `requirements.txt`, and once pandas & nose are installed they'll be efficiently cached together with the rest of Python, etc.